.github/workflows/ci.yml

```diff
@@ -218,6 +218,7 @@ jobs:
           env: |
             JAVA_HOME: /jvm
             JAVA_OPTS: "-Djdk.lang.Process.launchMechanism=vfork"
+            GITHUB_ENV: "/tmp/github_env"
```

Comment:
So, when starting the QEMU vm, all GitHub actions env variables are passed on to it, see here https://github.com/uraimo/run-on-arch-action/blob/a8003307a739516fdd80ee6d3da8924db811b8da/src/run-on-arch.sh#L101-L126
However the `GITHUB_ENV` var points to a file, which is not mapped inside the QEMU vm, so when trying to write to it it will fail, e.g. see [this comment](https://github.com/uraimo/run-on-arch-action/pull/53#issuecomment-1375690706).
So I override it with a dummy file `/tmp/github_env`, which kind of simulates the env file for mill when it calls `setShouldPublish` and [writes to that file](https://github.com/VirtusLab/scala-cli/blob/8ba8022069b2f35ff6374a8aeeb8f252cbc190cd/project/publish.sc#L213).